README.markdown

```diff
@@ -1904,6 +1904,8 @@ Note that the following API functions are currently disabled within this context
 * Subrequest API functions (e.g., [ngx.location.capture](#ngxlocationcapture) and [ngx.location.capture_multi](#ngxlocationcapture_multi))
 * Cosocket API functions (e.g., [ngx.socket.tcp](#ngxsockettcp) and [ngx.req.socket](#ngxreqsocket)).
 
+Note that Nginx may reject a request early with `ngx.HTTP_BAD_REQUEST`, `ngx.HTTP_REQUEST_TIMEOUT`, `ngx.HTTP_INTERNAL_SERVER_ERROR`, `ngx.HTTP_REQUEST_URI_TOO_LARGE`, `ngx.HTTP_NOT_IMPLEMENTED`, `ngx.HTTP_NOT_ALLOWED`, `ngx.HTTP_NOT_IMPLEMENTED` or `ngx.HTTP_REQUEST_HEADER_TOO_LARGE`. This causes variables set during the rewrite phase to be uninitialized in the logging phase.
```

Comment:
Better avoid using ngx_lua's own constants here. Please use `400 (Bad Request)` and etc here instead.